packages/label/src/Label.tsx

```diff
@@ -0,0 +1,33 @@
+import { Color } from '@littlespoon/theme/lib/colors'
+import type React from 'react'
+
+import { LabelBase } from './LabelBase'
+
+export interface LabelProps extends React.HTMLAttributes<HTMLSpanElement> {
+  /**
+   * The content of the component.
+   */
+  children?: React.ReactNode
+
+  /**
+   * The Category of the component. Defaults to "status".
+   */
+  category?: 'status' | 'product'
+
+  /**
+   * The background color of the component. Defaults to "success20".
+   */
+  color?: Color
+
+  /**
+   * The size of the component. Defaults to "medium".
+   */
+  size?: 'small' | 'medium' | 'large'
+}
+
+/**
+ * Label
+ */
+export default function Label({ size = 'medium', color = 'success20', ...other }: LabelProps) {
```

Comment:
Based on [thread](https://little-spoon.slack.com/archives/C02BQKELLEL/p1660162933987999?thread_ts=1660162302.829309&cid=C02BQKELLEL), design would like to default `color` to `transparent`

Reply:
Updated.